samples/bluetooth/direct_test_mode/src/dtm.h

```diff
@@ -15,358 +15,265 @@
+	bool aod_1us_rx;
 
-	/* Restore front-end module (FEM) default parameters (antenna, gain, delay). */
-	FEM_DEFAULT_PARAMS_SET = 6
+	/** Support for AoA reception 1 us switching and sampling */
+	bool aoa_1us_rx;
 };
 
-/* DTM Packet Type field */
-enum dtm_pkt_type {
-	/* PRBS9 bit pattern */
-	DTM_PKT_PRBS9 = 0x00,
-
-	/* 11110000 bit pattern (LSB is the leftmost bit). */
-	DTM_PKT_0X0F = 0x01,
+/** @brief DTM transmit power */
+struct dtm_tx_power {
+	/** Actual power in dBm */
+	int8_t power;
 
-	/* 10101010 bit pattern (LSB is the leftmost bit). */
-	DTM_PKT_0X55 = 0x02,
+	/** Power at minimum level */
+	bool min;
 
-	/* 11111111 bit pattern for Coded PHY.
-	 * Vendor specific command for Non-Coded PHY.
-	 */
-	DTM_PKT_0XFF_OR_VS = 0x03,
+	/** Power at maximum level */
+	bool max;
 };
 
-/* DTM events */
-enum dtm_evt {
-	/* Status event, indicating success. */
-	LE_TEST_STATUS_EVENT_SUCCESS = 0x0000,
-
-	/* Status event, indicating an error. */
-	LE_TEST_STATUS_EVENT_ERROR = 0x0001,
+/** @brief Function for initializing DTM module.
+ *
+ * @return 0 in case of success or negative value in case of error.
+ */
+int dtm_init(void);
 
-	/* Packet reporting event, returned by the device to the tester. */
-	LE_PACKET_REPORTING_EVENT = 0x8000,
-};
+/** @brief Prepare DTM for setup
+ *
+ * @note This function should be called before setup functions.
+ * The function can be called once before a block of setup functions.
+ */
+void dtm_setup_prepare(void);
 
-/* DTM error codes */
-enum dtm_err_code {
-	/* Indicate that the DTM function completed with success. */
-	DTM_SUCCESS,
+/** @brief Reset the DTM state.
+ *
+ * The PHY is set to Bluetooth Low Energy 1M mode.
+ * The modulation index is set to standard.
+ * The CTE is turned off.
+ *
+ * @return 0 in case of success or negative value in case of error.
+ */
+int dtm_setup_reset(void);
 
-	/* Physical channel number must be in the range 0..39. */
-	DTM_ERROR_ILLEGAL_CHANNEL,
+/** @brief Set the PHY for DTM.
+ *
+ * @param[in] phy The phy to be used.
+ *
+ * @return 0 in case of success or negative value in case of error.
+ */
+int dtm_setup_set_phy(enum dtm_phy phy);
 
-	/* Sequencing error: Command is not valid now. */
-	DTM_ERROR_INVALID_STATE,
+/** @brief Set the modulation for DTM.
+ *
+ * @param[in] modulation The modulation to be used.
+ *
+ * @return 0 in case of success or negative value in case of error.
+ */
+int dtm_setup_set_modulation(enum dtm_modulation modulation);
 
-	/* Payload size must be in the range 0..37. */
-	DTM_ERROR_ILLEGAL_LENGTH,
+/** @brief Read supported BLE features.
+ *
+ * @return Features supported by the device.
+ */
+struct dtm_supp_features dtm_setup_read_features(void);
 
-	/* Parameter out of range (legal range is function dependent). */
-	DTM_ERROR_ILLEGAL_CONFIGURATION,
+/** @brief Read the maximum supported parameter value by DTM.
+ *
+ * @param[in] parameter Value to be read.
+ * @param[out] max_val The pointer to the maximum value.
```

Comment:
```suggestion
 * @param[out] max_val  The pointer to the maximum value.
```
All params' descriptions per function should be aligned to the same vertical line.